packages/ia-prototype-apps/apps/details-react/index.html

```diff
@@ -12,6 +12,8 @@
     <script src="https://archive.org/bookreader/BookReaderHelpers.js"></script>
     <script src="https://archive.org/bookreader/LendingFlow.js"></script>
     <script src="https://archive.org/bookreader/BookReaderJSIA.js"></script>
+    <-- turn on debugging, comment out for no debugging-->
```

Comment:
@mitra42 is this where you were able to successfully test?

Reply:
Sorry - just saw this comment after the call, no I have currently no way to test anything in IAUX, I'm more than happy to do so but not sure that we have anything there that is testable standalone.   There is a PR proposed yesterday on dweb-archive to allow that to use components from ia-components, if that works I'll be actively testing ia-components & using in my (live) code from there on.